### PR TITLE
Update contact instructions: announce chat, preference for issues

### PR DESCRIPTION
**Pull request by oprypin**

### CODE REVIEW

**Comment by oprypin** [comment]
The chat room is https://gitter.im/mkdocs/community
FYI @Andre601 @pawamoy @mondeja  @ktomk

**Comment by Andre601** [comment]
> The chat room is https://gitter.im/mkdocs/community FYI @Andre601 @pawamoy @mondeja @ktomk

And I should care because......?

**Comment by Andre601** [comment]
> My guess is that this was a benevolent mention to you since you're an active contributor and commenter of the project and that's a proposed place for us to discuss easily.
> 
> There are other and certainly more gentle ways to state if you dislike those mentions or think that's inappropriate mate.
> 
> Thank you (kindly) 😸

My message certainly wasn't bad, angry or evil. It was a genuine question why I was mentioned here.
And not everyone can or will use IRC-like chats like gitter, especially when Discord is a much better alternative for that matter. But that's preference.

File: README.md [hunk 1]
```diff
@@ -27,11 +27,18 @@ user guide.
 
 If you need help with MkDocs, do not hesitate to get in contact with us!
 
-You can either open a [Discussion] to ask questions or create an [Issue] to
-report a bug or make a feature request. Please note that we may only provide
+- For questions and high-level discussions, use **[Discussions]** on GitHub.
+    - For small questions, a good alternative is the **[Chat room]** on
+      Gitter/Matrix (**new!**)
```
[the remark this answers is not in the document]
Comment (Andre601) [reply README.md:32]: I agree on this one. Having a "New" indicator always adds the extra work of removing it at one point in the future and it isn't really clear when this should be.

The text wouldn't really change without the "new" text so I would vote for removing it.

[the remark this answers is not in the document]
Comment (oprypin) [reply README.md:32]: Sorry, I'm just going to disagree there. I want to bring attention to it. And I am not scared to take up the task of removing the "new" eventually.

[the remark this answers is not in the document]
Comment (Andre601) [reply README.md:32]: I mean, people aren't *that* dumb to not notice a new place to communicate. You could also make like a discussion post announcing this.
And adding a "NEW!" label to this feels like all those companies trying to get you to pay for their "new" product/service. Like a desperate attempt of getting people to join this place.

[the remark this answers is not in the document]
Comment (oprypin) [reply README.md:32]: I will mention it on the GitHub release, but haven't decided about release notes on the site

**Pull request merged**